Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeHidParsers.cpp

```diff
@@ -0,0 +1,149 @@
+#include "pch.h"
```

Comment:
missing the license header

Reply:
fixed, as well as other files in library that were missing headers